packages/web-components/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@microsoft/fast-components-msft",
```

Comment:
Currently, per the issue in reference this package has been published already and there are projects which depend on this both internally at Microsoft and externally, so we can't change the name without consequence. The long term plan is to have this live under the fluent namespace, but that will require planning and some coordination. While I agree in principle of having the name match the package, that's really only possible if we want to leverage the `fast` namespace here; though, I think that would cause more confusion. I'll ping @levithomason as FYI here as we discussed the organization under "web-components" the other day.

Reply:
@ecraig12345 I think we need to work on integrating releases. We will likely need to prioritize that discussion, but in the meantime - are we able to release manually within the scope of the package? I have permissions to publish to this package at that namespace.

Reply:
Another question here - if we need to publish separately from the current system, do I need to opt out of the change requests to pass the build? If so, is that acceptable to do as part of this PR?